### ISSUE TITLE
[Bug]: annoying warning in server log

### ISSUE THREAD

**Issue by danielpil** [Fuzss/leavesbegone#9]
### Mod Loader (Required)

Fabric

### Minecraft Version(s) (Required)

1.21.3

### Mod Version(s) (Required)

1.21.3

### Minimal Setup (Required)

Yes

### Notes (Required)

i run a fabric server for myself and a few friends with purely QOL mods. in 1.21.3 it had Lithium, Ferritecore, and this mod (+ dependencies), as well as a datapack that removed work cost from anvils. when I updated my server to 1.21.4 earlier today and started playing, i noticed that the following error was getting spammed in the console: Unknown attachment type leavesbegone:random_block_ticks found when deserializing, skipping
this shouldnt be possible since the only thing i carried over to the updated server folder was the world file. everything else was freshly installed. ive attached a log where the spamming of this error got very annoying

### latest.log (Required)

https://gist.github.com/danielpil/311f20377dca7f5f1b9e6a5e34554597

**Comment by IsaacShoebottom** [comment]
I also get this, even with the mod removed

**Comment by meyverick** [comment]
Uninstalled this mod and it's now spamming.

```java
[14:09:32] [c2me-worker-0/WARN]: Unknown attachment type leavesbegone:random_block_ticks found when deserializing, skipping
[14:09:54] [Worker-Main-1/WARN]: Unknown attachment type leavesbegone:random_block_ticks found when deserializing, skipping
[14:09:54] [Worker-Main-2/WARN]: Unknown attachment type leavesbegone:random_block_ticks found when deserializing, skipping
```

**Comment by wadupubg** [comment]
> Uninstalled this mod and it's now spamming.
> 
> [14:09:32] [c2me-worker-0/WARN]: Unknown attachment type leavesbegone:random_block_ticks found when deserializing, skipping
> [14:09:54] [Worker-Main-1/WARN]: Unknown attachment type leavesbegone:random_block_ticks found when deserializing, skipping
> [14:09:54] [Worker-Main-2/WARN]: Unknown attachment type leavesbegone:random_block_ticks found when deserializing, skipping

have you fixed this? i have the same issue

**Comment by meyverick** [comment]
> > Uninstalled this mod and it's now spamming.
> > [14:09:32] [c2me-worker-0/WARN]: Unknown attachment type leavesbegone:random_block_ticks found when deserializing, skipping
> > [14:09:54] [Worker-Main-1/WARN]: Unknown attachment type leavesbegone:random_block_ticks found when deserializing, skipping
> > [14:09:54] [Worker-Main-2/WARN]: Unknown attachment type leavesbegone:random_block_ticks found when deserializing, skipping
> 
> have you fixed this? i have the same issue

No, I haven't managed to solve this error and the dev seems not responding to us.

I have chosen to remove it, install another one and report this one to Modrinth.

I think the only solution is cutting down every tree that were existing while the mod was installed.

**Comment by Fuzss** [comment]
This is just a warning message, there is nothing to be worried about.
The warning is only shown once for each chunk when it is first loaded after Leaves Be Gone has been removed.

**Comment by meyverick** [comment]
The problem is that if your map is very large, the message never stop. I mean that, after 3 weeks, I still got the message and it still spam my console.

**Comment by Fuzss** [comment]
There is nothing that can be done about it on my end, Fabric Api prints the message. 
Ideally it wouldn't serialize empty attachment data, that's why the data is stored in every chunk even when not necessary.
Your best option would be to get a mod that allows for filtering log messages.
